knative-example/README.md

```diff
@@ -0,0 +1,75 @@
+# Overview
+
+Sample javascript application implementing the classic [2048 game](https://en.wikipedia.org/wiki/2048_(video_game). Main project is based on the [game-2048 library](https://www.npmjs.com/package/game-2048) and [Webpack](https://webpack.js.org).
+
+Main purpose is to serve as a demo for the [DOKS-CI-CD](https://github.com/digitalocean/container-blueprints/tree/main/DOKS-CI-CD) blueprint.
+
+## Building the 2048 Game Application
+
+Main project is [npm](https://www.npmjs.com) based, hence you can build the application via:
+
+```shell
+npm install --include=dev
+
+npm build
+```
```

Comment:
Npm build fails for me using npm version 8.9.0.
This should be changed to `npm run build`

Reply:
Thanks. Fixed in latest commit.